test/karma.conf.js

```diff
@@ -166,7 +166,7 @@ module.exports = function setKarmaConfig(config) {
           os: 'OS X',
           os_version: 'Catalina',
           browser: 'chrome',
-          browser_version: '84.0',
+          browser_version: '90.0',
```

Comment:
Considering all the [test failures](https://app.circleci.com/pipelines/github/mui-org/material-ui/50818/workflows/2baf1c01-a7eb-47f8-ad28-99d83a37185c/jobs/285757) we may want to hold off on changing this until we figured out what changed.

Reply:
Ok, so we should revert back to `84.0` right.

Reply:
Currently bisecting what's possible. Will push a fix.